include/radio_types.h

```diff
@@ -298,7 +305,8 @@ typedef enum radio_cell_info_type_1_5 {
 
 /* Cast guint8 RadioCellIdentity_1_5.cellIdentityType to this. */
 typedef enum radio_cell_identity_type_1_5 {
-    RADIO_CELL_IDENTITY_1_5_GSM = 0,
+    RADIO_CELL_IDENTITY_1_5_NONE = 0,
+    RADIO_CELL_IDENTITY_1_5_GSM,
```

Comment:
That shifts the entire enum, are you sure?

Reply:
In the generated Android header (hardware/radio/1.5/types.h) there is noinit = 0 and gsm has value 1 in the CellIdentity hidl_discriminator.

Reply:
ok then, just double-checking!